### DISCUSSION TITLE
refer to table without ormar model

### DISCUSSION THREAD

**Discussion by avico78** [collerek/ormar#268] [Q&A]
Is it possible to fetch data using ormar where there's no ormar model configured?

in sqlalchemy  we are doing something like:
table_x= db.Table('non_ormar_table', metadata_ps_dvdrental, autoload=True, autoload_with=engine_ps_dvdrental)
print(table_x.columns.keys())

is it possible to refer to a table using ormar without define a model for it?
if yes, can u please refer me to documenaiton as i check it and couldn't find.

**Comment by collerek** [comment]
In sqlalchemy that gives you a Table (core element) and not a Model (orm element). 
So you still can use it but cannot refer to it from models?

What do you want to do with this table? 
As you still can do the same with sqlalchemy and run the query through databases (that's pretty much what ormar is doing under the hood).
